tests/test_package_manifest.py

```diff
@@ -14,6 +14,10 @@
 EXAMPLES_RAW_URL = "https://raw.githubusercontent.com/ethpm/ethpm-spec/master/examples"
 
 
+def test_can_generate_schema():
+    PackageManifest.schema()
```

Comment:
please use assertions next time in your tests

Reply:
The only assertion I had was that it successfully generates a schema without failing, so it can be used downstream in FastAPI